core/src/main/nl/tudelft/ti2806/riverrush/domain/event/TeamProgressEvent.java

```diff
@@ -11,7 +11,12 @@
 public class TeamProgressEvent implements Event {
 
     private int teamID;
-    private int progress;
+    private double progress;
+
+    public TeamProgressEvent(int team, double progres) {
```

Comment:
Change constructor to not require arguments otherwise we can't attach the event to the event dispatcher